### DISCUSSION TITLE
编辑模式下如何用css模拟空行？

### DISCUSSION THREAD

**Discussion by jxjxjhjh** [obsidianzh/forum#67] [Q&A常见问题]
我对所见即所得的唯一需求就是每行文字间的大间距，我想用css实现，可一直摸不到门道。
例如一般网页中，文字是
<p>111</p>
<p>111</p>
只要用css写
p{margin-bottom:20px}
即可。
那ob的编辑模式下（实际上typora我也尝试过，但找不到办法），如何实现？
很明显并不是用<p>来换行的。

**Comment by renmu123** [comment]
对行内框用 `line-height` 来增加间距
